rest/handler.go

```diff
@@ -226,13 +226,12 @@ func (h *handler) logRequestLine() {
 	if !base.LogEnabled("HTTP") {
 		return
 	}
-	as := h.currentEffectiveUserNameAsUser()
 	proto := ""
 	if h.rq.ProtoMajor >= 2 {
 		proto = " HTTP/2"
 	}
 
-	base.LogTo("HTTP", " #%03d: %s %s%s%s", h.serialNumber, h.rq.Method, base.SanitizeRequestURL(h.rq.URL), proto, as)
+	base.LogToR("HTTP", " #%03d: %s %s%s%s", h.serialNumber, h.rq.Method, base.SanitizeRequestURL(h.rq.URL), proto, base.UD(h.currentEffectiveUserNameAsUser()))
```

Comment:
I'm wondering if we should tag the request URL as UserData too.

Reply:
Yep, agreed